.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,28 @@
+name: golangci-lint
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
```

Comment:
We should remove all the comments, if we need something it should be a URL to the documentation of the action instead.

Personally, I would prefer nothing searching for the action if changes are needed are better, since then we get to the repo with the correct information, adding these as comments sets us in a position to have outdated "helpers"